pandas/core/sorting.py

```diff
@@ -186,7 +187,9 @@ def maybe_lift(lab, size) -> tuple[np.ndarray, int]:
     return out
 
 
-def get_compressed_ids(labels, sizes: Shape) -> tuple[np.ndarray, np.ndarray]:
+def get_compressed_ids(
+    labels, sizes: Shape
+) -> tuple[npt.NDArray[np.int64], npt.NDArray[np.int64]]:
```

Comment:
according to docstring

```suggestion
) -> tuple[npt.NDArray[np.intp], npt.NDArray[np.int64]]:
```